docs/source/faq.rst

```diff
@@ -5,6 +5,64 @@ This page contains answers to several frequently asked technical questions
 about Calico. It is updated on a regular basis: please check back for more
 information.
 
+What is Calico?
+---------------
```

Comment:
We have 'What' already now.  I think it would make sense to change this FAQ to 'Why Calico?'